### PR TITLE
Update to Nodejs 16 and fix the dependency issue

### PR DESCRIPTION
**Pull request by JamesJHPark**
This PR is to update the Nodejs version 14 to 16 and fix the dependency issue.

### CODE REVIEW

**Review by alolita — APPROVED** [review]

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -19,7 +19,6 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: '14'
```
Comment (anuraaga) [inline .github/workflows/pr.yml:22]: Let's keep this but set to 16

File: src/plugins/contributorsPlugin/gatsby-node.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Author: Patrick Arminio <patrick.arminio@gmail.com>
+// Licensed under the MIT License (MIT), see
```
Comment (anuraaga) [inline src/plugins/contributorsPlugin/gatsby-node.js:2]: Instead of a separate license file, you can just inline the license text here

Comment (JamesJHPark) [reply src/plugins/contributorsPlugin/gatsby-node.js]: I have inlined the license text here. Thank you.

File: .github/workflows/pr.yml (hunk 1 above)
Comment (JamesJHPark) [reply .github/workflows/pr.yml:22]: This has been fixed.

**Review by anuraaga — APPROVED** [review]

**Pull request merged**